### PR TITLE
WP-9041 - merge latest from singerio

### PR DESCRIPTION
**Pull request by yi-varicent**

### CODE REVIEW

File: tap_oracle/sync_strategies/common.py [hunk 1]
```diff
@@ -27,13 +27,21 @@ def send_schema_message(stream, bookmark_properties):
...
 def row_to_singer_message(stream, row, version, columns, time_extracted):
     row_to_persist = ()
     for idx, elem in enumerate(row):
         property_type = stream.schema.properties[columns[idx]].type
+        property_format = stream.schema.properties[columns[idx]].format
         if elem is None:
             row_to_persist += (elem,)
+        elif ('string' in property_type or property_type == 'string') and property_format == 'singer.decimal':
+            if len(str(elem)) > MAX_DECIMAL_DIGITS:
+                elem = elem.normalize()
```
Comment (yi-varicent) [inline tap_oracle/sync_strategies/common.py:43]: @pggagnon blame says you did something similar here, does this look right to you?  is the test just having a very long string?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
 # Changelog
 
+## 1.1.2
```
Comment (yi-varicent) [inline CHANGELOG.md:3]: @kchan-varicent can we just use this version #?  or you want a new version number?  ours is like 0.3.8

Comment (kchan-varicent) [reply CHANGELOG.md:3]: maybe 1.1.3? it really doesn't matter as long as we're increasing.

**Review by kchan-varicent — APPROVED** [review]

File: tap_oracle/sync_strategies/common.py (hunk 1 above)
Comment (pggagnon) [reply tap_oracle/sync_strategies/common.py:43]: Just saw this @yi-varicent I'm on the blame for a bunch of lines due to some formatting changes, so I don't have much explanation to provide. My actual change was just line 24.

**Pull request merged**